tpcx_bb/queries/load_test/tpcx_bb_load_test.py

```diff
@@ -50,7 +50,7 @@ def read_csv_table(table, chunksize="256 MiB"):
     names, types = get_schema(table)
     dtype = {names[i]: types[i] for i in range(0, len(names))}
 
-    data_dir = config["data_dir"]
+    data_dir = config["data_dir"].split('parquet_')[0]
```

Comment:
This assumes that the path we pass will have `parquet_xgb` like structure.   We might want to eventually add a `raw_dir` config to arg parser or something.

Reply:
Yes it does. Good thinking, let's do it in a followup as for now we're still tied the config file and I've been thinking we should restructure it anyway to make it easier on Slurm systems.